third_party/move/move-binary-format/src/access.rs

```diff
@@ -219,6 +219,15 @@ pub trait ModuleAccess: Sync {
         self.as_module().version
     }
 
+    fn num_immediate_dependencies(&self) -> usize {
+        // We need to subtract one for self handle.
```

Comment:
this is a bit scary, is there a test that ensures module_handles contains itself?

Reply:
Addressed in #14402.